float64 flight_duration
float64 double_support_duration
float64 step_height
float64 step_length
float64 com_amplitude
float64 com_phase
float64 trunk_angle
float64 am_task_weight
float64 torque_task_weight